core/src/com/unciv/models/ruleset/Building.kt

```diff
@@ -200,6 +197,147 @@ class Building : NamedStats(), IConstruction {
+                link="Building/$providesFreeBuilding")
+        }
+
+        val stats = this.clone()
+        val percentStats = getStatPercentageBonuses(null)
+        val specialists = newSpecialists()
+        if (uniques.isNotEmpty() || !stats.isEmpty() || !percentStats.isEmpty() || this.greatPersonPoints != null || specialists.isNotEmpty())
+            textList += FormattedLine()
+
+        if (uniques.isNotEmpty()) {
+            if (replacementTextForUniques.isNotEmpty())
+                textList += FormattedLine(replacementTextForUniques)
+            else
+                for (unique in getUniquesStrings()) textList += FormattedLine(unique)
+        }
+
+        if (!stats.isEmpty()) {
+            textList += FormattedLine(stats.toString())
+        }
+
+        if (!percentStats.isEmpty()) {
+            if (percentStats.production != 0f)
+                textList += FormattedLine(percentStats.production.formatSignedInt() + "% {Production}")
+            if (percentStats.gold != 0f)
+                textList += FormattedLine(percentStats.gold.formatSignedInt() + "% {Gold}")
+            if (percentStats.science != 0f)
+                textList += FormattedLine(percentStats.science.formatSignedInt() + "% {Science}")
+            if (percentStats.food != 0f)
+                textList += FormattedLine(percentStats.food.formatSignedInt() + "% {Food}")
+            if (percentStats.culture != 0f)
+                textList += FormattedLine(percentStats.culture.formatSignedInt() + "% {Culture}")
+        }
+
+        if (this.greatPersonPoints != null) {
+            val gpp = this.greatPersonPoints!!
+            if (gpp.production != 0f) textList +=
+                FormattedLine(gpp.production.formatSignedInt() + " " + "[Great Engineer] points".tr(),
+                    link="Unit/Great Engineer")
+            if (gpp.gold != 0f) textList +=
+                FormattedLine(gpp.gold.formatSignedInt() + " " + "[Great Merchant] points".tr(),
+                    link="Unit/Great Merchant")
+            if (gpp.science != 0f) textList +=
+                FormattedLine(gpp.science.formatSignedInt() + " " + "[Great Scientist] points".tr(), 
+                    link="Unit/Great Scientist")
+            if (gpp.culture != 0f) textList +=
+                FormattedLine(gpp.culture.formatSignedInt() + " " + "[Great Artist] points".tr(), 
+                    link="Unit/Great Artist")
+        }
+
+        if (specialists.isNotEmpty()) {
+            for ((specialistName, amount) in specialists)
+                textList += FormattedLine("+$amount " + "[$specialistName] slots".tr())
+        }
+
+        if (requiredNearbyImprovedResources != null) {
+            textList += FormattedLine()
+            requiredNearbyImprovedResources!!.withIndex().forEach {
+                textList += FormattedLine(
+                        (if (it.index == 0) "Requires worked" else "or") +
```

Comment:
U wot m8
There's no way this will be translated well.

Reply:
![image](https://user-images.githubusercontent.com/63000004/124838020-4ae40000-df86-11eb-8e48-69ab3014f2b4.png)
The 'von' is @Mape6 's work methinks, and the only point this sounds ugly to us krauts. "Benötigt bewirtschaftete ressource [x]" would read nicer - so, yes, these are a challenge because they limit the structural redistribution liberties of a translator severely. But a) I had no better idea at the moment, b) I prioritized nice english a little and c) Unciv in german is horrible anyway because you can't provide for grammar, noun gender, sometimes plural/singular is not fixed, etc. It's a compromise anyway.

Any better idea welcome, but pushes will have to wait a little - I need to drive a few thousand miles after work.